Restructuring/projects handout (#414)
neuefische/web-exercises: sessions/recap-project-1/quiz-app/README.md

- Create a remote repository and connect it to the local repository.
- Create the `index.html` and `styles.css` files.
- Ensure that the CSS file is loaded in the HTML document.

## Resources

> Download the require [Icons](https://lucide.dev/icons/) and save it in an "assets" folder within your app's main directory.

Reviewer: ```suggestion
Download the required [Icons](https://lucide.dev/icons/) and save them in an "assets" folder within your app's main directory.
```

- Update the navbar so that the bookmark icon is highlighted.
- Link the two pages via the anchor elements. (Also on the homepage).

> ## Bonus: Profile Page

Reviewer: ```suggestion
### Bonus: Profile Page
```

Author: It was decided to keep the Bonus as h2.
Task are mandatory feature, meanwhile bonus and bonus, extra

> 💡 The counter and switch don't have to work for now, we will implement this later.

> ## Recommendation

Reviewer: ```suggestion
## Recommendations
```
